packages/itwinui-css/src/workflow-diagram/workflow-diagram.scss

```diff
@@ -11,7 +11,7 @@
 @mixin iui-workflow-diagram-step {
   --_iui-workflow-diagram-circle-background-color: var(--iui-color-background-positive);
   --_iui-workflow-diagram-circle-border-color: var(--iui-color-background-positive);
-  --_iui-workflow-diagram-circle-border-radius: 0;
+  --_iui-workflow-diagram-circle-border-radius: #{$iui-border-radius};
   --_iui-workflow-diagram-circle-text-color: var(--iui-color-foreground-accessory);
   --_iui-workflow-diagram-track-before-color: var(--iui-color-foreground-positive);
   --_iui-workflow-diagram-track-after-color: var(--iui-color-foreground-positive);
```

Comment:
I was trying to find where this was used. I found its use in `workflow-diagram.html` but I couldn't find any component called `workflow-diagram` or a way to access this page from the website (no button). Typing this link manually gave a 404 error (https://itwin.github.io/iTwinUI/workflow-diagram.html). How do I access this `workflow-diagram.html` page?

Reply:
workflow diagram is not present in the main branch. you can find the deployment for this branch at https://itwin.github.io/iTwinUI/746/workflow-diagram.html

and if you're running this branch locally, it should also show up in the list.